examples/boilerplate/360-video-post/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
```

Comment:
Can you move this example to `examples/test/post-processing` and rename the titles/metatags to match?